### PR TITLE
minised: update to 1.16

### PR DESCRIPTION
**Pull request by mtboehlke**
#### Testing the changes
- I tested the changes in this PR: **briefly**

Short description changed to comply with linting

### CODE REVIEW

File: srcpkgs/minised/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-revision=2
+version=1.16
+revision=1
 build_style=gnu-makefile
-short_desc="A smaller, cheaper, faster sed implementation"
+short_desc="Smaller, cheaper, faster sed implementation"
 maintainer="Richard Taityr <dicktyr@yahoo.co.uk>"
 license="BSD-3-Clause"
 homepage="http://exactcode.com/opensource/minised/"
 distfiles="http://dl.exactcode.de/oss/${pkgname}/${pkgname}-${version}.tar.gz"
```
Comment (ericonr) [inline srcpkgs/minised/template:10]: At least homepage works with https, distfiles probably do as well.

Comment (mtboehlke) [reply srcpkgs/minised/template]: Yep, https worked just fine

**Comment by mtboehlke** [comment]
One issue I found while building this outside of xbps-src is that the run script for the tests, as of this version, contains a bash-ism, but the shebang is still /bin/sh.  It will fail with dash as /bin/sh, but works in the chroot. I was planning on reporting upstream.

**Comment by ericonr** [comment]
No need to patch, shell inside xbps-src will always be bash. Reporting upstream sounds good tho!

**Pull request merged**